test/phase_info_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import openhtf
+
+
+def PlainFunc():
+  """Plain Docstring"""
+  pass
+
+
+def ExtraArgFunc(input=None):
+  return input
+
+
+class TestPhaseInfo(unittest.TestCase):
+
+  def testBasics(self):
+      phase = openhtf.PhaseInfo.WrapOrCopy(PlainFunc)
+      assert phase.func is PlainFunc
```

Comment:
pytest has a smart assert system in place that can convert this to a well-documented output, negating the need for `self.assert*`. I can switch to those for consistency with other files, but pytest provides a lot of features that shouldn't be lost just because they're not available internally